### PR TITLE
Add the file 'services.py'

### PR DESCRIPTION
**Pull request by ceciliarivero**

### CODE REVIEW

File: shareprogress/services.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Managing Buttons
+def create_button(page_url, button_template, page_title=None, *args):
+    # page_url: The URL of the page to be shared.
```
Comment (EricBoersma) [inline shareprogress/services.py]: One change I'd make here:

`#` will do a single-line comment in Python, whereas `"""` (3 double quotes in a row) will do a doc-block comment, which is designed for holding exactly these kinds of comments (and tends to get read by software which auto generates documents). The content of these looks great, though, I'd just condense them into a single doc-block.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,7 @@
 .idea/*
 *.iml
+.Python
+bin/
+include/
+lib/
+.DS_Store
```
Comment (EricBoersma) [inline .gitignore]: Make sure to add a newline at the end of the file so github doesn't complain :)

**Comment by EricBoersma** [comment]
I like the structure here, but one thing we need to include are functions to interact with pages, too - for this library, I made sure to clear space for us to handle the whole API from SP, not just the pieces that we need, so we'll want to include pages and (I think) ways to get the data back out of the API, too.

That said, these are great function stubs and a really good first step. :+1:

File: shareprogress/services.py (hunk 1 above)
Comment (ceciliarivero) [inline shareprogress/services.py]: Done! (Thought I'm wondering if I need to delete the lines between each argument explanation)

File: .gitignore (hunk 2 above)
Comment (ceciliarivero) [inline .gitignore]: Done!

File: shareprogress/services.py (hunk 1 above)
Comment (EricBoersma) [inline shareprogress/services.py]: I'd check out this link here for convention: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

File: shareprogress/services.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+def update_button(id, *args):
+    """
+    id: button ID.
+    
+    *args: Received arguments for the variant/s.
+    we would use the update_variant() function here.
+    """
+
+def delete_button(id):
+    # id: button ID.
```
Comment (EricBoersma) [inline shareprogress/services.py]: Make sure to add `pass` back into this function.

File: shareprogress/services.py (hunk 1 above)
Comment (ceciliarivero) [inline shareprogress/services.py]: Interesting to know!
I changed the style of the Docstrings, does it look OK?

**Comment by EricBoersma** [comment]
Looks good! Merging!

**Pull request merged**